packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -608,9 +608,10 @@ describe("wrangler", () => {
       );
       await runWrangler("init");
       expect(fs.existsSync("./tsconfig.json")).toBe(true);
-      const tsconfigJson = JSON.parse(
-        fs.readFileSync("./tsconfig.json", "utf-8")
-      );
+      // tsconfig.json isn't necessarily valid json, so let's use eval to parse it
+      const tsconfigJson = eval(
```

Comment:
![image](https://user-images.githubusercontent.com/18808/155127594-16bf95ae-cb78-41e7-9042-8ef3040864b0.png)

Reply:
We can use:

```ts
import {parseConfigFileTextToJson} from "typescript";
...
parseConfigFileTextToJson("./tsconfig.json", fs.readFileSync("./tsconfig.json", "utf-8"));
```

Reply:
I considered it... heck why not. changing.